hieradata/class/ci_master.yaml

```diff
@@ -70,6 +70,7 @@ govuk_jenkins::config::create_agent_role: true
 govuk_jenkins::config::executors: '0'
 
 govuk_jenkins::job_builder::jobs:
+  - govuk_jenkins::jobs::deploy_app_downstream
   - govuk_jenkins::jobs::integration_deploy
```

Comment:
How come we have both integration_deploy and deploy_app_downstream?

Reply:
(I'm assuming integration_deploy is for puppet and licensify) - Is it possible to make the intentions of these two job clearer?

Reply:
I agree this isn't a great outcome for this change. How about we split the `integration_deploy` job into its component jobs? I'd like to do that as a separate PR, though, if that's OK with you.